frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/select_option_cell/extension.dart

```diff
@@ -108,12 +110,17 @@ class SelectOptionTag extends StatelessWidget {
         mainAxisSize: MainAxisSize.min,
         children: [
           Flexible(
-            child: FlowyText.medium(
-              name,
-              fontSize: FontSizes.s11,
-              overflow: TextOverflow.ellipsis,
-              color: AFThemeExtension.of(context).textColor,
-            ),
+            child: FlowyText.medium(name,
+                fontSize: FontSizes.s11,
+                overflow: TextOverflow.ellipsis,
+                color: ColorAnalyzer().getAppropriateTextColor(
+                    color,
+                    Theme.of(context)
+                        .extension<AFThemeExtension>()!
+                        .textColorAlt,
+                    Theme.of(context)
+                        .extension<AFThemeExtension>()!
+                        .textColor)),
```

Comment:
I already mentioned that you should use `AFThemeExtension.of(context)`, please correct.